packages/launcher/src/renderer/launcher/wallets/WalletAddLedgerModal.js

```diff
@@ -94,65 +142,141 @@ export default class WalletAddLedgerModal extends Component<Props, State> {
       variables: { input, userID: this.props.userID },
       onCompleted: (response, errors) => {
         if (errors || !response) {
+          // console.log(errors)
           const error =
             errors && errors.length ? errors[0] : new Error(MUTATION_ERR_MSG)
+
           this.displayError(error)
         } else {
-          this.props.onClose()
+          this.props.onSuccess ? this.props.onSuccess() : this.props.onClose()
         }
       },
       onError: err => {
+        // console.log(err)
```

Comment:
Could this line be removed completely rather than commented out please?

Reply:
oh.. sure... my bad,...

Reply:
done.